physionet-django/training/models.py

```diff
@@ -0,0 +1,144 @@
+from django.db import models
+
+from project.modelcomponents.fields import SafeHTMLField
+
+
+class Course(models.Model):
+    training_type = models.ForeignKey('user.TrainingType',
+                                      on_delete=models.CASCADE, related_name='courses')
+    version = models.FloatField(default=1.0)
+
+    class Meta:
+        default_permissions = ('change',)
+        unique_together = ('training_type', 'version')
```

Comment:
Docs say this may be deprecated soon - https://docs.djangoproject.com/en/4.2/ref/models/options/#unique-together

You can switch to UniqueConstraint


```suggestion
    constraints = [
        models.UniqueConstraint(fields=['training_type', 'version'], name='unique_course')
    ]
```